pennylane/transforms/decompositions/clifford_t_transform.py

```diff
@@ -454,7 +454,7 @@ def circuit(x, y):
             decomp_ops.append(qml.GlobalPhase(phase))
 
     # Construct a new tape with the expanded set of operations
-    new_tape = type(tape)(decomp_ops, compiled_tape.measurements, shots=tape.shots)
+    new_tape = compiled_tape.copy(operations=decomp_ops)
```

Comment:
```suggestion
    new_tape = tape.copy(operations=decomp_ops, measurements=compiled_tape.measurements)
```

Reply:
`compiled_tape` comes from `qml.compile`, which has already copied `tape.shots` into `compiled_tape`, so there is no need to be copying the original tape here.